app/views/layout.erb

```diff
@@ -15,6 +15,7 @@
 		<!-- Jquery -->
 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js">
```

Comment:
Please remove this line! 
Jquery is already being used in our project! Check bottom of this file.